src/components/organisms/Carousel/index.js

```diff
@@ -1,18 +1,10 @@
 import React from 'react';
-import CarouselText from '../../molecules/CarouselText';
-import Button from '../../atoms/Button';
+import Slider from '../../molecules/Slider';
```

Comment:
move slider to orgarnisms, since its more organism than molecule

Reply:
Moved